### ISSUE TITLE
Turbo-flask not able to push() on Raspberry Pi

### ISSUE THREAD

**Issue by HFMan** [miguelgrinberg/turbo-flask#29]
I can't get Push() to work when running the example demo on a Raspberry Pi (Debian Linux, stretch release).  Python 3.8.13, Flask 2.0.3, Werkzeug 2.1.1. 

turbo.can_push returns false when client is using Chrome or Edge.  The same exact code running on Windows with same versions of Python, Flask, and Turbo-Flask works as expected (and identical Chrome and Edge clients).

Any idea what might be causing this?

**Comment by miguelgrinberg** [comment]
Are both tests establishing a websocket connection? That is probably what the difference is. If the test based on the RPi does not make a WebSocket connection, why is that? The client should have attempted a connection. Did it? If it did, why did it fail?

**Comment by HFMan** [comment]
Miguel,

I don't know exactly how to determine if a websocket connection has been
made.  No errors or any indication that a websocket connection failed was
found in the server console output or anywhere else.  Any suggestions on
how to make this determination?

**Comment by HFMan** [comment]
Okay- a bit more information.  Using Chrome developer tools-> Network Tab,  I have verified that the RPi does not make a WebSocket connection to my Chrome browser (but no errors indicated on the server side).  I do see my HTML page in the 
 client browser, but it does not dynamically update.  

OTOH, running the same exact code on Windows Python, the same Chrome browser with developer tools enabled shows a WebSocket connection and it all works as expected.

Thoughts?

**Comment by HFMan** [comment]
Okay, now I understand.  After you suggestion, via Chrome tools I determined that the rPi server was not serving up the script<> tag that defines the websocket that the client should be connecting to.  So, that told me the Turbo object was not instantiating correctly.  Alas- I discovered a small typo in my code0 where I instantiate the Turbo object, I had written {{ turbo }} as opposed to {{ turbo() }}.  Small detail that was easy to miss, yet catastrophic.   After fixing this, all is working properly.  Thanks for your swift replies to this issue!
